src/Common/AllocatorWithMemoryTracking.h

```diff
@@ -32,7 +33,7 @@ struct AllocatorWithMemoryTracking
         size_t bytes = n * sizeof(T); /// NOLINT(bugprone-sizeof-expression)
         auto trace = CurrentMemoryTracker::alloc(bytes);
 
-        T * p = static_cast<T *>(malloc(bytes));
+        T * p = static_cast<T *>(je_malloc(bytes));
```

Comment:
Double memory accounting. We would love to call the "pure" malloc function without tracking. 

But according to this file: https://github.com/ClickHouse/ClickHouse/blob/master/contrib/jemalloc-cmake/include/jemalloc/jemalloc_rename.h#L6-L31 the `je_malloc` is a macro that is replaced by `malloc` by preprocessor and thus also intercepted. 

Maybe I can call the `__real_malloc` 🤔